src/sentry/static/sentry/app/components/events/contexts/operatingSystem/getOperatingSystemKnownDataDetails.tsx

```diff
@@ -0,0 +1,55 @@
+import {t} from 'app/locale';
+import {defined} from 'app/utils';
+
+export enum OperatingSystemKnownDataDetailsType {
```

Comment:
Does this make sense to be in its own seperate file - considering it is being exported to be used somewhere else?

Reply:
... maybe it is better to create a `types.ts` file .. Let's do that :)